pkg/config/config.go

```diff
@@ -264,6 +268,11 @@ func (c *Config) ConfigFileUsed() string {
 	return c.base.ConfigFileUsed()
 }
 
+// Paths returns the paths to the configuration files merged
```

Comment:
nit: the comment should also indicate the config paths returned were applied in order from lower index to higher index, i.e. the last config file in the list overrides all others.

Reply:
added in https://github.com/bacalhau-project/bacalhau/pull/4488/commits/918ea588bbd75999996e5fca3ec790e94ff171b3